go.mod

```diff
@@ -13,4 +13,4 @@ require (
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/resty.v1 v1.12.0
 	gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7 // indirect
-)
+)
```

Comment:
You should be able to ignore these changes with git, even though I can't tell what the difference it. Is it some whitespace issue?

Reply:
The go mod and sum contain a new line at the end of the file. My editor removed these, but I added them again.

Reply:
Nice, that would explain it